usecases/steg_asymptotic_2sensors/virtual_experiment_asymptotic_model_data.yaml

```diff
@@ -0,0 +1,15 @@
+MatB: [17.0, 40.4473684210526, 63.89473684210526, 87.34210526315792, 110.78947368421052,

```

Comment:
Please remove th yaml files from the git repo, they can always be regenerated on the fly. In particular, if you change someting in the input, you would have to commit the changes to the yaml files, which is suboptimal and error prone.